slices/libgdiplus.yaml

```
@@ -0,0 +1,27 @@
+package: libgdiplus
+
+essential:
+  - libgdiplus_copyright
+
+slices:
+  libs:
+    essential:
+      - libc6_libs
+      - libcairo2_libs
+      - libexif12_libs
+      - libfontconfig1_libs
+      - libgif7_libs
+      - libglib2.0-0t64_libs
+      - libharfbuzz0b_libs
+      - libjpeg-turbo8_libs
+      - libpango-1.0-0_libs
+      - libpangocairo-1.0-0_libs
+      - libpng16-16t64_libs
+      - libtiff6_libs
+      - libx11-6_libs
+    contents:
+      /usr/lib/libgdiplus.so.0*:
```

```suggestion
      /usr/lib/libgdiplus.so:
```
Otherwise the `./usr/lib/libgdiplus.so` path would not be installed.

---

You're missing the glob.

---

Ah, danke!

---

This needs to be resolved by the way. :)